packages/truffle-core/lib/develop.js

```diff
@@ -20,18 +20,16 @@ var Develop = {
       chainPath = path.join(__dirname, "../", "chain.js");
     }
 
-    var cmd = spawn("node", [chainPath, ipcNetwork, JSON.stringify(options)], {
+    return spawn("node", [chainPath, ipcNetwork, JSON.stringify(options)], {
       detached: true,
       stdio: "ignore"
     });
-
-    callback();
   },
 
   connect: function(options, callback) {
```

Comment:
Can't get rid of this callback?

Reply:
Is this an old comment from before I removed this callback?